python/pyspark/ml/wrapper.py

```diff
@@ -16,6 +16,10 @@
 #
 
 from abc import ABCMeta, abstractmethod
+import sys
```

Comment:
Oh, good catch thanks!  I wasn't using the def for `basestring` but I still am iterating the array/list with `xrange`.  I was thinking it would be possible for someone to use a large list, like when using a vocabulary.  I suppose I could use `enumerate` if you think that's better?